browser/ui/views/tabs/brave_tab_search_button.h

```diff
@@ -17,12 +17,17 @@ class BraveTabSearchButton : public TabSearchButton {
   BraveTabSearchButton(const BraveTabSearchButton&) = delete;
   BraveTabSearchButton& operator=(const BraveTabSearchButton&) = delete;
 
+  void disable_fill_color() { disable_fill_color_ = true; }
```

Comment:
this name is really confusing to me, where is this called from?

Reply:
`set_fill_color_disabled` would be better imo

Reply:
Yes, obviously. Done.